docs/content/docs/tasks_usage/traffic_management/demos/gloo_edge_ingress_demo.md

```diff
@@ -0,0 +1,125 @@
+    $ kubectl get pods -n gloo-system
+    NAME                             READY   STATUS    RESTARTS   AGE
+    discovery-b7c89f698-7xpw5        1/1     Running   0          173m
+    gloo-7b844d6cd4-djnlk            1/1     Running   0          173m
+    ingress-7ffcc9df95-7fb6j         1/1     Running   0          173m
+    ingress-proxy-76cf8c6bdb-m727w   1/1     Running   0          173m
+    ```
+
+1. Install OSM onto the cluster.
+    ```bash
+    osm install
+    ```
+
+1. Deploy the `httpbin` service into the `httpbin` namespace after enrolling its namespace to the mesh. The `httpbin` service runs on port `14001`.
+    ```bash
+    # Create the httpbin namespace
+    kubectl create namespace httpbin
+
+    # Add the namespace to the mesh
+    osm namespace add httpbin
+
+    # Deploy httpbin service in the httpbin namespace
+    kubectl apply -f docs/example/manifests/samples/httpbin/httpbin.yaml -n httpbin
+    ```
+
+    Confirm the `httpbin` service and pods are up and running.
+
+    ```console
+    $ kubectl get pods -n httpbin
+    NAME                       READY   STATUS    RESTARTS   AGE
+    httpbin-74677b7df7-7f5v4   2/2     Running   0          149m
+    ```
+
+    ```console
+    $ kubectl get svc -n httpbin
+    NAME      TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)     AGE
+    httpbin   ClusterIP   10.0.115.107   <none>        14001/TCP   149m
+    ```
+
+
+1. Apply an ingress configuration yaml to expose the HTTP path `/status/200` on the `httpbin` service with `kubectl apply -f`
+    ```yaml
+    apiVersion: networking.k8s.io/v1beta1
+    kind: Ingress
+    metadata:
+    name: httpbin-ingress
+    namespace: httpbin
+    annotations:
+        kubernetes.io/ingress.class: gloo
+    spec:
+    rules:
+    - host: httpbin.httpbin.svc.cluster.local
+        http:
+        paths:
+        - path: /status/200
+            backend:
+            serviceName: httpbin
+            servicePort: 14001
+    ```
+
+
+1. Confirm that the httpbin-ingress has been successfully deployed.
+
+    ```console
+    $ kubectl get ingress -n httpbin
+    NAME              CLASS    HOSTS                               ADDRESS         PORTS   AGE
+    httpbin-ingress   <none>   httpbin.httpbin.svc.cluster.local   52.234.160.38   80      152m
+    ```
+
+1. Configure the `Upstream` object to use OSM's root ca bundle: 
+
+    ```yaml
+    apiVersion: gloo.solo.io/v1
+    kind: Upstream
+    metadata:
+    name: httpbin-httpbin-14001
+    namespace: gloo-system
+    spec:
+    sslConfig:
+        sni: "httpbin.httpbin.svc.cluster.local"
+        secretRef:
+        name: osm-ca-bundle
+        namespace: gloo-system
+    kube:
+        selector:
+        app: httpbin
+        serviceName: httpbin
+        serviceNamespace: httpbin
+        servicePort: 14001
+    ```
+
+1. Confirm that a request to the httpbin service from the external IP address of the Ingress resource succeeds (in this case, the external address would be `52.234.160.38`)
```

Comment:
Suggest change it to `a request to the the httpbin service via the external IP address of ...`.

My first impression was send request from ingress to httpbin